Automattic/jetpack
# Crowdsignal shortcode: implement review suggestions

Implement review suggestions from #29307 that arrived after the auto-merge.

## Proposed changes:
* Implement review suggestions from #29307 that arrived after the auto-merge.

### Other information:

- N/A Have you written new tests for your changes, if applicable?

## Jetpack product discussion
None. See #29307 for the review suggestions that are implemented here.

## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:
Same as #29307:

- Set up your site with a jQuery-less theme like `twentytwentyone`
- Enable infinite scrolling
- Create a page with a Crowdsignal shortcode for a poll
- While logged out, load the page in your browser, with devtools open
- Ensure that jQuery does not load
- Ensure that the poll loads correctly
- Load `/` or any other infinite scrolling page for your site
- If necessary, scroll until the post with the poll loads
- Ensure that jQuery does not load
- Ensure that the poll loads correctly

You can also repeat the steps above for a theme that does use jQuery, in order to ensure that things don't break there.
